curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/63ff732ad5c56c1a497c7f16.md

```diff
@@ -0,0 +1,138 @@
+---
+id: 63ff732ad5c56c1a497c7f16
+title: Step 6
+challengeType: 0
+dashedName: step-6
+---
+
+# --description--
+
+Apart from the `.then()` method, the `.catch()` method is another asynchronous JavaScript method with which you can handle errors. This is useful in case the `Promise` gets rejected.
+
+Chain a `.catch()` to the last `.then()`. Set the error variable to `err` inside the `catch()`. Add your callback, then using `console.error()`, log that error to the console with the text `There was an error: ${err}`. Since you're using the `err` variable in the text, don't forget to use template literal (\`\`) instead of single or double quotes.
+
+**Note**: Now you can terminate the code with a semi-colon. You can't do that in the previous steps because you'll signal to JavaScript to stop parsing your code, which will affect your `fetch()` syntax.
```

Comment:
```suggestion
The `.catch()` method is another asynchronous JavaScript method you can use to handle errors. This is useful in case the `Promise` gets rejected.

Chain `.catch()` to the last `.then()`. Pass in `err` as the parameter and a callback function. Inside the callback, use `console.error()` to log possible errors to the console with the text `There was an error: ${err}`. Since you're using `err` in the text, don't forget to use a template literal string with backticks (\`\`) instead of single or double quotes.

**Note**: Now you can terminate the code with a semicolon. You can't do that in the previous steps because you'll signal to JavaScript to stop parsing your code, which will affect the `fetch()` syntax.
```